src/event-listeners/onDocumentLoaded.js

```diff
@@ -12,12 +12,10 @@ export default dispatch => () => {
   dispatch(actions.setDocumentLoadingProgress(1));
   dispatch(actions.setWorkerLoadingProgress(1));
 
-  setTimeout(() => {
-    dispatch(actions.closeElement('progressModal'));
-    dispatch(actions.resetLoadingProgress());
-    dispatch(actions.resetUploadProgress());
-    dispatch(actions.setIsUploading(false));
-  }, 300);
+  dispatch(actions.closeElement('progressModal'));
```

Comment:
I feel like we need the 300ms timeout. If we remove this I see that the progress bar is not filled before closing...

Reply:
put back in the 300 ms timeout. ran end to end tests again. they aren't affected.